CHANGELOG.md

```diff
@@ -10,8 +10,9 @@ Please also have a look at our
 
 ### Added
 
-- Partial support for CSS Color Module Level 4 syntax:
+- Partial support for CSS Color Module Level 4:
   - `rgb` and `rgba`, and `hsl` and `hsla` are now aliases (#797}
+  - Parse color functions that use the "modern" syntax
```

Comment:
```suggestion
  - Parse color functions that use the "modern" syntax (#800)
```

Reply:
I'd intended to do this once I'd determined what the PR number would be, but then forgot.  Now added.